pandapower/networks/lv_schutterwald.json

```diff
@@ -290,7 +290,7 @@
+      "_object": "{\"columns\":[\"name\",\"std_type\",\"hv_bus\",\"lv_bus\",\"sn_mva\",\"vn_hv_kv\",\"vn_lv_kv\",\"vk_percent\",\"vkr_percent\",\"pfe_kw\",\"i0_percent\",\"shift_degree\",\"tap_side\",\"tap_neutral\",\"tap_min\",\"tap_max\",\"tap_step_percent\",\"tap_step_degree\",\"tap_pos\",\"tap_phase_shifter_type\",\"parallel\",\"df\",\"in_service\"],\"index\":[0,1,3,4,5,6,7,10,11,12,13,14,15,16],\"data\":[[\"T_idx_47\",\"0.4 MVA 20\\/0.4 kV\",2983,3000,0.4,20.0,0.4,6.0,1.425,1.35,0.3375,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_45\",\"0.25 MVA 20\\/0.4 kV\",2984,3001,0.25,20.0,0.4,6.0,1.44,0.8,0.32,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_35\",\"0.4 MVA 20\\/0.4 kV\",2986,3003,0.4,20.0,0.4,6.0,1.425,1.35,0.3375,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_77\",\"0.4 MVA 20\\/0.4 kV\",2987,3004,0.4,20.0,0.4,6.0,1.425,1.35,0.3375,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_78\",\"0.63 MVA 20\\/0.4 kV\",2988,3005,0.63,20.0,0.4,6.0,1.206,1.65,0.2619,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_119\",\"0.63 MVA 20\\/0.4 kV\",2989,3006,0.63,20.0,0.4,6.0,1.206,1.65,0.2619,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_118\",\"0.63 MVA 20\\/0.4 kV\",2990,3007,0.63,20.0,0.4,6.0,1.206,1.65,0.2619,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_117\",\"0.4 MVA 20\\/0.4 kV\",2993,3010,0.4,20.0,0.4,6.0,1.425,1.35,0.3375,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_80\",\"0.63 MVA 20\\/0.4 kV\",2994,3011,0.63,20.0,0.4,6.0,1.206,1.65,0.2619,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_73\",\"0.63 MVA 20\\/0.4 kV\",2995,3012,0.63,20.0,0.4,6.0,1.206,1.65,0.2619,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_43\",\"0.63 MVA 20\\/0.4 kV\",2996,3013,0.63,20.0,0.4,6.0,1.206,1.65,0.2619,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_81\",\"0.63 MVA 20\\/0.4 kV\",2997,3014,0.63,20.0,0.4,6.0,1.206,1.65,0.2619,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_71\",\"0.4 MVA 20\\/0.4 kV\",2998,3015,0.4,20.0,0.4,6.0,1.425,1.35,0.3375,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true],[\"T_idx_ZUSATZ\",\"0.63 MVA 20\\/0.4 kV\",2999,3016,0.63,20.0,0.4,6.0,1.206,1.65,0.2619,150.0,\"hv\",0,-2,2,2.5,0.0,1,false,1,1.0,true]]}",
```

Comment:
these "false" in the 4th to last column are still "false" from the old bool "is_phase_shifter". Now these are used as false (=0) for "tap_phase_shifter_type". This is not correct. (incidently the indirect conversion: false - 0 - ratio tap changer leads to the same result, but by pure luck. For "true" it would be wrong, because before it meant ideal phase shifter, now it means symmetrical tap changer.)
Better to leave the files in the old format with the column "is_phase_shifter", then convert using the function "convert_format" to new column names and correct values.

Reply:
This concerns all json files